src/tasks/netSdk/netSdkTaskUtils.ts

```diff
@@ -33,25 +33,20 @@ export type RidCpuArchitecture =
 
 export const NetSdkRunTaskType = 'dotnet-container-sdk';
 const NetSdkDefaultImageTag = 'dev'; // intentionally default to dev tag for phase 1 of this feature
-const ErrMsgNoWorkplaceFolder = l10n.t(`Unable to determine task scope to execute task ${NetSdkRunTaskType}. Please open a workspace folder.`);
 
 export async function getNetSdkBuildCommand(context: DockerTaskExecutionContext): Promise<string> {
 
     const configuration = 'Debug'; // intentionally default to Debug configuration for phase 1 of this feature
-    const projPath = await inferProjPath(context.actionContext, context.folder);
-
     // {@link https://github.com/dotnet/sdk-container-builds/issues/141} this could change in the future
-    const publishFlag = NetCoreTaskHelper.isWebApp(projPath) ? '-p:PublishProfile=DefaultContainer' : '/t:PublishContainer';
-
-    const folderName = await quickPickWorkspaceFolder(context.actionContext, ErrMsgNoWorkplaceFolder);
+    const publishFlag = NetCoreTaskHelper.isWebApp(context.folder.uri.fsPath) ? '-p:PublishProfile=DefaultContainer' : '/t:PublishContainer';
 
     const args = composeArgs(
         withArg('dotnet', 'publish'),
         withNamedArg('--os', await normalizeOsToRidOs()),
         withNamedArg('--arch', await normalizeArchitectureToRidArchitecture()),
         withArg(publishFlag),
         withNamedArg('--configuration', configuration),
-        withNamedArg('-p:ContainerImageName', getValidImageName(folderName.name), { assignValue: true }),
+        withNamedArg('-p:ContainerImageName', getValidImageName(context.folder.name), { assignValue: true }),
```

Comment:
Change to ContainerRepository since ContainerImageName is deprecated in the image builder targets that will ship with the SDK

Reply:
was not aware of this, good point